dashboard/src/components/TestDetails/TestDetails.tsx

```diff
@@ -134,6 +149,56 @@ const TestDetailsSections = ({
     [setSheetType],
   );
 
+  const regressionSection: ISection = useMemo(() => {
+    return {
+      title: formatMessage({ id: 'testDetails.statusHistory' }),
+      rightIcon: (
+        <TooltipIcon
+          tooltipId="testDetails.statusHistoryTooltip"
+          tooltipValues={{ amount: test.status_history.length }}
+          iconClassName="size-5"
+          contentClassName="whitespace-pre-line"
+        />
+      ),
+      subtitle: (
+        <div>
+          <Tooltip>
+            <TooltipTrigger>
+              <Badge variant="blueTag">
+                {capitalizeString(test.regression_type)}
```

Comment:
You could possibly achieve the same effect just using [CSS text-transform](https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform). If you don't need the actual string to be capitalized for some reason (since this only change the way its shown to the user and not the string itself), than I believe this is better than creating a function for this

```suggestion
              <Badge className="capitalize" variant="blueTag">
                {test.regression_type}
```

Reply:
I was using the regression_type as a string in the field value, so that's why I made the capitalize function, but now that it is in a component I can use the css class, you're right